### ISSUE TITLE
when i excute train.py, i meet this problem : loss = inf.

### ISSUE THREAD

**Issue by collins43** [amdegroot/ssd.pytorch#162]
when i excute train.py, i meet this problem : loss = inf.
then i find loc_loss = inf.Can anyone help me,thanks!

**Comment by haochenheheda** [comment]
i meet the same problem

**Comment by Pythonisfun** [comment]
so am i

**Comment by julycetc** [comment]
Me too, what's the problem?

**Comment by shufanwu** [comment]
what you can try is decreasing the lr or increasing the batch size

**Comment by HubJuice** [comment]
This is because log calculation in box_utils.py.
In function "encode", you can add epsilon for preventing explosion. 
For example,
    eps = 1e-5
    g_wh = (matched[:, 2:] - matched[:, :2]) / priors[:, 2:]
    g_wh = torch.log(g_wh+eps) / variances[1]

**Comment by indrajitsg** [comment]
This should be merged with the master branch
